docs/v8-migration-api.md

```diff
@@ -307,6 +307,10 @@ Check out the [v8-migration guide](v8-migration.md) for other user-level changes
 
   __Migration__: Adapt to the new behaviour. You can use `NodePath#shouldSkip` to check whether a NodePath has been skipped before calling `NodePath#requeue()`.
 
+- Remove methods starting with `_` ([#16504](https://github.com/babel/babel/pull/16504))
```

Comment:
Can you prepare a list of removed `_`-starting methods here? It will help others reach this page from search engines.

Reply:
Thanks. Nit: Can you sort it alphabetically?